extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -372,7 +393,39 @@ public function getCuratedContentItems( $items ) {
 	 * @param $item
 	 * @return mixed
 	 */
-	public function processCuratedContentItem( $item ) {
+	public function processCuratedContentItem( $item, $newFormat=false ) {
+		if ( $newFormat ) {
+			$result = [];
+			$result[ 'label' ] = empty($item['label']) ? $item['title'] : $item['label'];
```

Comment:
What do you think about using Null Coalescing Operator (http://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) ?